orbstation/code/customisation/quirks/linguist.dm

```diff
@@ -0,0 +1,10 @@
+//Linguist - lets you learn an extra language. From Skyrat.
+/datum/quirk/linguist
+	name = "Linguist"
+	desc = "You're a student of numerous languages and come with an additional language point."
+	value = 4
+	mob_trait = QUIRK_LINGUIST
+	gain_text = span_notice("Your brain seems more equipped to handle different modes of conversation.")
+	lose_text = span_danger("Your grasp of the finer points of Draconic idioms fades away.")
```

Comment:
I don't know if I like either of these implying that you have a _brain_ determined to be better at languages, or referencing a specific one you might not have on loss.
Losing this quirk is also weird because you won't unlearn a language, but I don't think it can happen outside of admin intervention anyway.

Reply:
I've edited the text here, removed the mob trait, and removed the medical record text as well. I don't feel like it's particularly relevant to your doctor that you speak four languages instead of three.